### PR TITLE
Feature/css structure

### PR DESCRIPTION
**Pull request by AliaksandraNovikava**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="../css/styles.css" />
+    <link rel="stylesheet" href="../css/global.css" />
     <script src="../js/index.js" defer></script>
```
Comment (vaupunkt) [inline index.html]: I think one "." should be enough :)
```suggestion
    <link rel="stylesheet" href="./css/global.css" />
    <script src="./js/index.js" defer></script>
```

File: index.html [hunk 2]
```diff
@@ -68,23 +68,23 @@ <h2 class="card__question">What property flips the axes in flexbox?</h2>
...
                 alt="bookmark"
             /></a>
           </li>
           <li>
             <a href="../profile/index.html"
               ><img
-                class="footer-nav-icon"
+                class="footer-nav__icon"
                 src="../assets/user-solid.svg"
                 alt="homepage icon"
```
Comment (vaupunkt) [inline index.html]: same as on top. One "." should also do it :)
```suggestion
              ><img
                class="footer-nav__icon"
                src="./assets/house-solid.svg"
                alt="homepage icon"
            /></a>
          </li>
          <li>
            <a href="./bookmarks/index.html"
              ><img
                class="footer-nav__icon"
                src="./assets/bookmark-solid.svg"
                alt="bookmark"
            /></a>
          </li>
          <li>
            <a href="./profile/index.html"
              ><img
                class="footer-nav__icon"
                src="./assets/user-solid.svg"
                alt="homepage icon"
```

File: index.html [hunk 3]
```diff
@@ -68,23 +68,23 @@ <h2 class="card__question">What property flips the axes in flexbox?</h2>
           <li class="active">
             <a href="/index.html"
```
Comment (vaupunkt) [inline index.html:69]: to be consistent, you could add a "." here
```suggestion
            <a href="./index.html"
```

File: components/stats/stats.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  text-align: center;
+  position: relative;
+  box-shadow: var(--box-shadow);
+}
+.stats img {
+  width: 15%;
+}
+
+.stats h3,
+.settings h3 {
```
Comment (vaupunkt) [inline components/stats/stats.css]: should this go into the settings.css?

File: components/navigation/navigation.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  padding: 10px;
+  text-align: center;
+}
+.footer-nav__icon {
+  height: 30px;
+  filter: var(--primary-color-filter);
+}
+.footer-nav__icon:hover,
+.footer-nav .active {
+  filter: var(--secondary-color-filter);
```
Comment (vaupunkt) [inline components/navigation/navigation.css:18]: you could put this :hover on the .footer-nav li - element, so works for the whole button, not online by hovering over the icon

File: components/navigation/navigation.css [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  width: 33.3%;
+  padding: 10px;
+  text-align: center;
+}
+.footer-nav__icon {
+  height: 30px;
+  filter: var(--primary-color-filter);
+}
+.footer-nav__icon:hover,
+.footer-nav .active {
```
Comment (vaupunkt) [inline components/navigation/navigation.css:17]: this could be .footer-nav__icon--active, I think? :)

**Review by vaupunkt — APPROVED** [review]
looks very good so far and i only have some small comments

File: index.html (hunk 1 above)
Comment (AliaksandraNovikava) [reply index.html]: I copied this way of writing from Andrea. But you're right. I should correct it 👍

File: components/stats/stats.css (hunk 4 above)
Comment (AliaksandraNovikava) [reply components/stats/stats.css]: You're right. I overlooked it 😅

File: index.html (hunk 1 above)
Comment (AliaksandraNovikava) [reply index.html]: I've decided to write without the dots.

File: components/navigation/navigation.css (hunk 5 above)
Comment (AliaksandraNovikava) [reply components/navigation/navigation.css:18]: I wanted only the icon to change the color.

File: components/navigation/navigation.css (hunk 6 above)
Comment (AliaksandraNovikava) [reply components/navigation/navigation.css:17]: You're right. I forgot to apply the BEM method on this element.

**Pull request merged**